_posts/2015-09-04-channel.html

```diff
@@ -1902,6 +1904,230 @@
+				<td>Requested page size (50 by default, max 50)</td>
+			</tr>
+			<tr>
+				<td>detail_level</td>
+				<td>string</td>
+				<td>OPTIONAL</td>
+				<td>Verbosity level. Possible value: "minimal". In case of minimal verbosity level the result set is limited to id, and links data. If the channel is protected (see <a href="channel.html#security">security</a> section), only minimal data can be retrieved without a valid access token.</td>
+			</tr>
+		</table>
+
+		<h6>Response</h6>
+		<p>A list of key-value pairs under a "videos" key.</p>
+		<p>⚠️ The "media_urls" field below is being deprecated. For video download URLs please see the <a href="#downloading-videos_28">Downloading videos</a> section. ⚠️</p>
+		<p>Example:</p>
+		<pre><code class="json">{
+"videos": [{
+		"id": "111",
+		"title": "Title of video",
+		"description": "Description of video",
+		"url": "http://www.ustream.tv/recorded/111",
+		"length": "12345.123456",
+		"created_at": 1399388736,
+		"file_size": "120000",
+		"views": 0,
+		"thumbnail": {
+			"default": "https://ustvstaticcdn1-a.akamaihd.net/111.jpg"
+		},
+		"media_urls": {
+			"flv": "http://tcdn.ustream.tv/video/111"
+		},
+		"links": {
+			"channel": {
+				"href": "https://api.ustream.tv/channels/1.json",
+				"id": "1"
+			}
+		}
+	},
+	...
+	]
+}</code></pre>
+
+		<p>Minimal response example:</p>
+		<pre><code class="json">{
+"video": {
+		"id": "111",
+		"links": {
+			"channel": {
+				"href": "https://api.ustream.tv/channels/1.json",
+				"id": "1"
+			}
+		}
+	}
+}</code></pre>
+
+		<p>Explanation of some values:</p>
+		<table class="table responsive">
+			<tr>
+				<th>key</th>
+				<th>type</th>
+				<th width="100%">Description</th>
+			</tr>
+			<tr>
+				<td>length</td>
+				<td>string</td>
+				<td>Video length in seconds</td>
+			</tr>
+			<tr>
+				<td>created_at</td>
+				<td>int</td>
+				<td>Video create date in UNIX timestamp</td>
+			</tr>
+			<tr>
+				<td>file_size</td>
+				<td>string</td>
+				<td>Video file size in bytes</td>
+			</tr>
+		</table>
+
+		<p>The paging information can be found under the "paging" element. Example:</p>
+		<pre><code class="json">{
+"paging": {
+		"previous": "https://api.ustream.tv/channels/CHANNEL_ID/featured-videos.json?pagesize=PAGE_SIZE&page=PREVIOUS_PAGE",
+		"next": "https://api.ustream.tv/channels/CHANNEL_ID/featured-videos.json?pagesize=PAGE_SIZE&page=NEXT_PAGE",
+		"page_count": 8,
+		"item_count": 38
+	}
+}</code></pre>
+
+		<h6>Error codes</h6>
+		<table class="table responsive">
+			<tr>
+				<th>error value</th>
+				<th>HTTP response code</th>
+				<th width="100%">Error conditions</th>
+			</tr>
+			<tr>
+				<td>not_found</td>
+				<td>404 Not found</td>
+				<td>
+					<p>This error code is returned in the following cases:</p>
+					<ul>
+						<li>The channel does not exist.</li>
+						<li>The featured videos on the channel are not shown.</li>
+					</ul>
+				</td>
+			</tr>
+		</table>
+
+		<h3>Update channel featured videos</h3>
```

Comment:
"channel featured videos" helyett "featured videos" vagy "featured videos of the channel" jobb lenne